src/js/peteshow-controller.coffee

```diff
@@ -1,5 +1,6 @@
-_  = require('lodash')
-cs = require('calmsoul')
+_     = require('lodash')
+cs    = require('calmsoul')
+store = require('./peteshow-storage')
```

Comment:
In this case, we're definitely going to need it, but i'll pull it off for sanity sake.